Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -156,6 +213,16 @@ protected theorem IsNClique.map (h : G.IsNClique n s) {f : α ↪ β} :
   ⟨by rw [coe_map]; exact h.1.map, (card_map _).trans h.2⟩
 #align simple_graph.is_n_clique.map SimpleGraph.IsNClique.map
 
+theorem isNClique_map_iff [DecidableEq β] (hn : 1 < n) {t : Finset β} {f : α ↪ β} :
+    (G.map f).IsNClique n t ↔ ∃ (s : Finset α), G.IsNClique n s ∧ s.map f = t := by
```

Comment:
Not very important, but you wouldn't write `∀ (s : Finset α), ....`, would you?
```suggestion
    (G.map f).IsNClique n t ↔ ∃ s : Finset α, G.IsNClique n s ∧ s.map f = t := by
```
and elsewhere